cime_config/allactive/config_pesall.xml

```diff
@@ -1402,19 +1402,19 @@
       </pes>
    </mach>
   </grid>  
-  <grid name="a%ne30np4.pg2">
+  <grid name="a%ne30np4">
```

Comment:
The previous string was not matching our ne30_ne30 grid, which many e3sm_scream_v1 tests use. The long name for that grid is `a%ne30np4_l%ne30np4_oi%ne30np4_r%null_g%null_w%null_z%null_m%gx1v6`. I also had to bump node count for that grid from 2 to 4 as you can see below in order to avoid the memory error from YAKL.